CONTRIBUTING.md

```diff
@@ -0,0 +1,55 @@
+# Contributing
+
+## Prerequisites
+
+## Build actors and providers
+
+From the top-level **directory** build with `make`. This should complete
+without errors.
+
+## Configuration
+
+Update paths in file `deploy/env` to match your development environment.
+
+## Running
+
+The script `deploy/run.sh` contains commands to run everything. In the
+`deploy` folder, run `run.sh` to see a list of available subcommands.
+
+Start the local registry server, nats server, wasmcloud host,
+actors, and providers. If this is your first time running running this
+app, add `--console` to the end of the following command to open a new
+terminal window with the host logs. The logs may be useful for
+diagnosing any problems.
+
+```bash
+./run.sh all
+# or, to open a $TERMINAL window with host logs
+./run.sh all --console
+```
+
+After a successful startup the _washboard_ should look similar to the following screenshot:
```

Comment:
We should probably remove references to the OTP washboard and instead show output from `wash get inventory`

Reply:
yeeted washboard refs and added `wash get inventory`